studio/.env

```diff
@@ -6,13 +6,11 @@ DEFAULT_PROJECT_NAME=Default Project
 
 SUPABASE_URL=http://localhost:8000
 SUPABASE_PUBLIC_URL=http://localhost:8000
-NEXT_PUBLIC_SUPABASE_URL=$SUPABASE_URL
-NEXT_PUBLIC_SUPABASE_PUBLIC_URL=$SUPABASE_PUBLIC_URL
```

Comment:
Do we know if it's safe to remove these variables? Could some clients depend on this because it's public?

May be we can remove in a separate PR just in case.

Reply:
This is a revert from a PR made about 3 weeks ago. I don't think the variables are used, but I can split in a separate PR or eventually leave them there, they are not conflicting AFAIK